runner/daemon/src/mill/daemon/MillMain0.scala

```diff
@@ -247,7 +247,9 @@ object MillMain0 {
                             new MillBuildBootstrap(
                               projectRoot = BuildCtx.workspaceRoot,
                               output = out,
-                              keepGoing = config.keepGoing.value,
+                              // In BSP server, we want to evaluate as many tasks as possible,
+                              // in order to give as many results as available in BSP responses
+                              keepGoing = bspMode || config.keepGoing.value,
```

Comment:
While I agree this may make sense, we should decide what the best way is to handle defaults. We currently set many BSP specifics via CLI args (e.g. we set `--disable-ticker` and `--color false`). Here we implicitly change defaults for `keepGoing` which means a `--keep-going` is having no effect. We should either handle all defaults via CLI args or as derived default values, but not a mix of both.